lms/static/sass/discussion/utilities/_developer.scss

```diff
@@ -44,7 +44,7 @@ body.discussion {
 
       // alert copy
       .message {
-        @include font-size(12);
+        font-size: 12px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also 12px is tiny. 16px should be our minimum. If we absolutely need smaller, 14px.